services/authentication/package.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
I think we also need to add this as a workspace in the root package.json so dependency hoisting works correctly.

Reply:
Fixed

Reply:
🙏 Thanks!